### ISSUE TITLE
Zulip support?

### ISSUE THREAD

**Issue by uhlhosting** [mathew-fleisch/bashbot#123]
Hi,

Any chance to add zulip support?

**Comment by mathew-fleisch** [comment]
I don't think so. I hadn't heard of this app before, but looking over the api docs, it looks like the slack integration uses the webhook api, whereas Bashbot uses the Slack Socket API to establish a constant socket connection with slack servers. If you have experience, and know otherwise, I'd be happy to look over PRs to introduce support for it.
